src/Gui/Forms/MainFormInteractor.cs

```diff
@@ -317,6 +317,33 @@ public void AddMetadataFile()
             }
         }
 
+        /// <summary>
+        /// Prompts the user for a script module and adds to the project.
+        /// </summary>
+        public void AddScriptModule()
```

Comment:
Prefer 'script file' over 'script module'.

Reply:
Done in 9742be9